pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/model/defaultvalues/RgpProbandDataCreatorTest.java

```diff
@@ -0,0 +1,188 @@
+package org.broadinstitute.dsm.model.defaultvalues;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonParser;
+import org.broadinstitute.dsm.db.dao.settings.FieldSettingsDao;
+import org.broadinstitute.dsm.model.ddp.DDPActivityConstants;
+import org.broadinstitute.dsm.model.elastic.Activities;
+import org.broadinstitute.dsm.model.elastic.sort.MockFieldSettingsDao;
+import org.broadinstitute.dsm.util.TestUtil;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+
+public class RgpProbandDataCreatorTest {
+
+    private static final Logger logger = LoggerFactory.getLogger(RgpProbandDataCreatorTest.class);
+    private static List<Activities> activities;
+    private static String refSourceValues;
+    private static String refSourceDetails;
+    private static MockFieldSettingsDao mockDao;
+
+    @BeforeClass
+    public static void Setup() throws Exception {
```

Comment:
Should be **setup**
We have a checkstyles configuration file that might not be loaded in your IDE that can help find these things.  See pepper-apis/checkstyle.xml and pepper-apis/docs/setting-up-checkstyle.md

Reply:
I also get a blizzard of checkstyle errors when running tests, so this did not stand out.